src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
> It's not a class.
I don't care about this convention. I think it sucks to limit this convention for a construct I rarely use in JavaScript anyway. I don't use Classes/Prototypes.

Router is more distinctive. It's not a component, like in React, but they also use Router, which is familiar.